### ISSUE TITLE
(Random Entities) NBT tags for custom textures

### ISSUE THREAD

**Issue by LorenzoF06** [sp614x/optifine#2745]
It'd be a nice thing if it was possible to apply a texture to an entity only if it has certain NBT tags
For example, I am trying to do that if a minecart_chest has a custom name, it will get the texture
```
skins.2=2
nbt.2={CustomName:"{\"translate\":\"container.boat_chest\"}"}
```
Well, probably this is impossible also with the name, because the minecart_chest is made from chest and minecart, but I don't know if the name also applies to the minecart part, but anyways, that should be a thing.

**Comment by Francy-Chan** [comment]
Yup, I'm trying to edit a sign texture based on it's text, at first I thought that it's work like CIT, but I was wrong.
